### ISSUE TITLE
No CORS Policy

### ISSUE THREAD

**Issue by bofia** [phalt/swapi#104]
Endpoint doesn't have a Access-Control-Allow-Origin header, which prevents chrome from using this API

**Comment by miayam** [comment]
Yes, I can fetch data from Google Developer Tools (console), but it doesn't work on localhost.
